components/kubeflow/kfserving/src/kfservingdeployer.py

```diff
@@ -378,7 +383,8 @@ def main():
         inferenceservice_yaml=inferenceservice_yaml,
         watch_timeout=watch_timeout,
         min_replicas=min_replicas,
-        max_replicas=max_replicas
+        max_replicas=max_replicas,
+        request_timeout=request_timeout
```

Comment:
Can we keep the argument order consistent by moving `request_timeout` after argument `inferenceservice_yaml`?

Reply:
Done. 👍🏻